test/python/WMComponent_t/AgentStatusWatcher_t/AgentStatusWatcher_t.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+"""
+AgentStatusWatcher test
+"""
+
+from __future__ import print_function
+import threading
+import unittest
+from WMQuality.TestInitCouchApp import TestInitCouchApp as TestInit
+
+from WMComponent.AgentStatusWatcher.AgentStatusPoller import AgentStatusPoller
+
+
+class AgentStatusWatcher_t(unittest.TestCase):
+    """
+    TestCase for WorkQueueManagerTest module
+    """
+
+    def setUp(self):
+        """
+        setup for test.
+        """
+        self.testInit = TestInit(__file__)
+        self.testInit.setLogging()
+        self.testInit.setDatabaseConnection()
+        #self.testInit.setSchema(customModules=["WMCore.WMBS"],
```

Comment:
Can we delete it?

Reply:
The commented lines you mean? If so, yes. Will do.